### DISCUSSION TITLE
Post relationship dropped if item updated separately

### DISCUSSION THREAD

**Discussion by richard1st** [pods-framework/pods#6645] [Q&A]
**Describe the bug**
If you create an item within a relationship filed it links. But If you then edit that item it drops the relationship. 
**To Reproduce**
1. Create new pod "part order"
2. Add filed "part order" - Relationship [type: pick]	
› (shop_order) (Post Type)
3. Extend "shop_order" (woocommerce) 
4. Add filed "part_order" - Relationship [type: pick]	
› Parts Orders (part_order) (Post Type)
5. Go into Shop Order and Create part order. The part gets listed and saved on list after update.
6. Go into Part Order - The shop order ID shows in custom filed but not in list.
7. Update Part Order - Now the custom filed is cleared braking the relationship 

**Expected behavior**
Relationship items show on both post type lists. 
Relationship is still saved regardless of where changes to post are made

**Screenshots**
http://snpy.in/CipMPV
## Pods Version
Pods - Custom Content Types and Fields: 2.7.9
## WordPress Environment

<details>
```
WordPress Version: 4.9.8

PHP Version: 7.1.20-1+ubuntu16.04.1+deb.sury.org+1

MySQL Version: 5.5.5

Server Software: nginx/1.15.1

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1 Safari/605.1.15

Session Save Path: /var/lib/php/sessions

Session Save Path Exists: No

Session Save Path Writeable: No

Session Max Lifetime: 1440

Opcode Cache:

Apc: Yes
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: Yes
APCu: Yes
Memcache: No
Memcached: Yes
Redis: Yes
WPDB Prefix: wp_

WP Multisite Mode: Yes

WP Memory Limit: 64M

Pods Network-Wide Activated: Yes

Pods Install Location: /www/1111/public/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Storefront

Currently Active Plugins:

Admin Columns Pro: 4.3.7
Admin Columns Pro - Pods: 1.2
Admin Columns Pro - WooCommerce: 3.1.2
Admin Menu Editor Pro: 2.6.6
Advanced Woo Search PRO: 1.31
All-in-One WP Migration: 6.74
All-in-One WP Migration Google Drive Extension: 2.36
All-in-One WP Migration Multisite Extension: 3.56
AutomateWoo: 4.3.1
Booster Plus for WooCommerce: 3.8.0
Code Snippets: 2.10.1.1
Custom Admin Page by BestWebSoft: 0.1.6
Force First and Last Name as Display Name: 1.0
Gravity Forms: 2.3.3.4
Gravity Forms + Custom Post Types: 3.1.13
Gravity Forms Twilio Add-On: 2.5
Gravity Forms User Registration Add-On: 3.9.5
Loco Translate: 2.1.5
Members: 2.1.0
Network Username Restrictions Override: 1.2
Password Protected: 2.2.2
Pods - Custom Content Types and Fields: 2.7.9
Post Status Notifier: 1.9.1
White Label CMS: 1.6.4
WooCommerce: 3.4.5
WooCommerce Advanced Bulk Edit: 4.4.1
WooCommerce Customer History: 1.2.1
WooCommerce Force Sells: 1.1.15
WooCommerce Gravity Forms Product Add-Ons: 3.3.3
WooCommerce Mutlisite Reporting: 1.3
WooCommerce Order Status & Actions Manager: 2.3.7
WordPress Toolbar Editor: 1.2.2
WP Mail SMTP: 1.3.3
```
</details>

**Comment by quasel** [comment]
That's weird - 
In the configuration ist the bi-directional option fully setup? (field details, check on both sides)
Step 6 seems to be the most interesting part: 
- you see the custom field but nothing in the pods ui part? 
- is your screenshot done using admin columns? 
it seems when you edit the "config" isn't fully loaded from pods hence on save the connection is removed 
after Step 7 is the connection removed on Shop Order too? 

What happens if you start from part_order and assign  a shop_order? 

Side note: we tend to recommend to name fields that hold a relationship with rel_<target_pod> or similar for better identification ;)

**Comment by richard1st** [comment]
I think the settings are correct, as you can see in the screenshot. 
http://snpy.in/FeiRvi
- you see the custom field but nothing in the pods ui part?
Exactly. no shop_order is in the list
 - after Step 7 is the connection removed on Shop Order too?
No. Shop order still keeps relationship 
- What happens if you start from part_order and assign a shop_order?
it show new order in list but when you save, it disappears from the list. The relationship still shows in custom filed but if you update, it disappears again.

**Comment by quasel** [comment]
i see an possible issue your CPT and Field have the same name! - pleas try renaming the field `part_order` to e.g.  `rel_shop_order` we recommend relationships like that for more clarity :D when it comes to fields ;)

**Comment by richard1st** [comment]
I made the change as you can see in screenshot
http://snpy.in/PPzAyz
Still same behavior

**Comment by quasel** [comment]
@jimtrue do I miss something obvious here?
